src/components/Sidebar/contentTreeNode.js

```diff
@@ -93,7 +93,7 @@ const NestedContentTreeNode = styled(
 
 const NodeCollapseButton = styled(({ className, isCollapsed, collapse }) => {
   return (
-    <button onClick={collapse} aria-label="collapse" className={className}>
+    <button /*onClick={collapse}*/ aria-label="collapse" className={className}>
```

Comment:
If we don't need `onClick={collapse}`, please remove it. We don't need to use code comment.

Reply:
Sorry

Reply:
@mildronize  No worries, nothing wrong.